Makefile

```diff
@@ -99,6 +101,23 @@ $(MANPAGES_TROFF): $(BUILDDIR)/docs/%: $(SRCDIR)/docs/%.rst | $(BUILDDIR)/docs/
 .PHONY: man
 man: $(MANPAGES_TROFF)
 
+#
+# OpenAPI Code Generation
+#
+# The `openapi` targets allow regenerating the request handlers from the openapi
+# specifications. Note that these targets modify the source-tree and the generated
+# code is committed upstream.
+#
+
+OPENAPIS_YAML = $(wildcard $(SRCDIR)/openapi/*/openapi.yml)
+OPENAPIS_GO = $(patsubst $(SRCDIR)/openapi/%/openapi.yml,$(SRCDIR)/internal/%/openapi.go,$(OPENAPIS_YAML))
+
+$(OPENAPIS_GO): $(SRCDIR)/internal/%/openapi.go: $(SRCDIR)/openapi/%/openapi.yml
+	$(OAPI_CODEGEN) --generate "types,chi-server,spec" "$<" >"$@"
+
+.PHONY: openapi
+openapi: $(OPENAPIS_GO)
+
```

Comment:
You have leveled up your GNU-make skills! :+1: 

If the subdirs seem too convoluted, you could also do: `openapi/openapi-%.yml -> internal/%/openapi.go`
Or something similar.. Might be easier to manage than 1-file-directories.

Reply:
Good point, did something along those lines. Thanks!